tmux/eval_vad.yml

```diff
@@ -0,0 +1,64 @@
+#
+# Used to evaluate Voice Activity Detection (VAD) for a specified ROS bag of data.
+# This configuration should be run by itself (e.g. not in combination with
+# another tmuxinator launch).
+#
+
+name: VAD Evaluation
+root: <%= ENV["ANGEL_WORKSPACE_DIR"] %>
+
+# Optional tmux socket
+# socket_name: foo
+
+# Note that the pre and post options have been deprecated and will be replaced by
+# project hooks.
+
+# Project hooks
+
+# Runs on project start, always
+# on_project_start: command
+on_project_start: |
+  export ROS_NAMESPACE=${ROS_NAMESPACE:-/debug}
+  export CONFIG_DIR=${ANGEL_WORKSPACE_DIR}/src/angel_system_nodes/configs
+# Run on project start, the first time
+# on_project_first_start: command
+
+# Run on project start, after the first time
+# on_project_restart: command
+
+# Run on project exit ( detaching from tmux session )
+# on_project_exit: command
+
+# Run on project stop
+# on_project_stop: command
+
+# Runs in each window and pane before window/pane specific commands. Useful for setting up interpreter versions.
+# pre_window: rbenv shell 2.0.0-p247
+
+# Pass command line options to tmux. Useful for specifying a different tmux.conf.
+# tmux_options: -f ~/.tmux.mac.conf
+tmux_options: -f <%= ENV["ANGEL_WORKSPACE_DIR"] %>/tmux/tmux.conf
+
+# Change the command to call tmux.  This can be used by derivatives/wrappers like byobu.
+# tmux_command: byobu
+
+# Specifies (by name or index) which window will be selected on project startup. If not set, the first window is used.
+# startup_window: editor
+
+# Specifies (by index) which pane of the specified window will be selected on project startup. If not set, the first pane is used.
+# startup_pane: 1
+
+# Controls whether the tmux session should be attached to automatically. Defaults to true.
+# attach: false
+
+windows:
+  - vad: ros2 run angel_system_nodes voice_activity_detector --ros-args
+      -r __ns:=${ROS_NAMESPACE}
+      -p input_audio_topic:=HeadsetAudioData
+      -p output_voice_activity_topic:=DetectedVoiceData
+      -p vad_server_url:=http://communication.cs.columbia.edu:55667/vad
+      -p vad_cadence:=2
+      -p vad_margin:=0.25
+      -p max_accumulation_length:=10
+  # - ros_bag_play: ros2 bag play <<PATH_TO_BAG_FILE>>
+  - ros_bag_play: ros2 bag play /angel_workspace/ros_bags/rosbag2_2023_03_01-17_28_00/rosbag2_2023_03_01-17_28_00_0.db3
```

Comment:
Please replace the `ros_bag_play` window configurations prior to merging. Also, please re-arrange the ordering to be aligned with current convention.